app/authoring/testing.md

```diff
@@ -44,13 +44,16 @@ The most useful method when unit testing a generator is `helpers.run()`. This me
 ```js
 var path = require('path');
 
-beforeEach(function () {
+it('generate a project', function () {
   // The object returned acts like a promise, so return it to wait until the process is done
-  return helpers.run(path.join(__dirname, '../app'))
+  helpers.run(path.join(__dirname, '../app'))
```

Comment:
We need to add a `return` on this line so the test frameworks wait for the promise to resolve before marking the test as passing.

Reply:
Done.